### ISSUE TITLE
No config found even though one is passed

### ISSUE THREAD

**Issue by billywade** [textlint/textlint#580]
**What version of textlint are you using?**
v11.2.1

**What file type (Markdown, plain text, etc.) are you using?**
Org-mode, but I specify the extension in my config and is unrelated to my issue.

**What did you do? Please include the actual source code causing the issue.**
I ran the command `textlint --config ~/.textlintrc ~/Homework/Spring\ 2019/Linux\ Workstation\ and\ Server\ OS/Module4Assignment.org`.
**What did you expect to happen?**
Textlint to either lint the file and output the errors or say that no errors were found.
**What actually happened? Please include the actual, raw output from textlint.**
```

== No rules found, textlint hasn’t done anything ==

Possible reasons:
* Your textlint config file has no rules.
* You have no config file and you aren’t passing rules via command line.
* Your textlint config has a syntax error.

=> How to set up rules?
https://github.com/textlint/textlint/blob/master/docs/configuring.md
```
  textlint:cli Running on files, stdin-filename: undefined +0ms
  textlint:module-loader config Config {
  textlint:module-loader   configFile: undefined,
  textlint:module-loader   rulesBaseDirectory: undefined,
  textlint:module-loader   rules: [],
  textlint:module-loader   disabledRules: [],
  textlint:module-loader   filterRules: [],
  textlint:module-loader   disabledFilterRules: [],
  textlint:module-loader   presets: [],
  textlint:module-loader   plugins: [],
  textlint:module-loader   pluginsConfig: {},
  textlint:module-loader   rulesConfig: {},
  textlint:module-loader   filterRulesConfig: {},
  textlint:module-loader   rulePaths: [],
  textlint:module-loader   formatterName: undefined,
  textlint:module-loader   quiet: false,
  textlint:module-loader   color: true,
  textlint:module-loader   cache: false,
  textlint:module-loader   cacheLocation: '/home/user/git/yendor/.textlintcache' } +0ms

== No rules found, textlint hasn’t done anything ==

Possible reasons:
* Your textlint config file has no rules.
* You have no config file and you aren’t passing rules via command line.
* Your textlint config has a syntax error.

=> How to set up rules?
https://github.com/textlint/textlint/blob/master/docs/configuring.md
```
It's worth noting that the repo mentioned does not have a textlintcache file, it's just the directory I happened to be in at the time.
<!--
You can get debug log by running textlint with `--debug` option.
$ $(npm bin)/textlint --debug target.md
# Please paste the debug log to the issue or use http://gist.github.com/
--->

**Comment by azu** [comment]
> configFile: undefined,

Fmm, Debug output does not found `.textlintrc`.

Your `.textlinrc` file is invalid syntax as JSON file.
Maybe, this syntax error cause the issue.

Please remove reduplicated `}`.

```diff
{
    "rules": {
        "write-good": true
        "no-start-duplicated-conjunction": true
        "max-comma": true
        "terminology": true
        "period-in-list-item": true
        "abbr-within-parentheses": true
        "alex": true
        "common-misspellings": true
        "en-max-word-count": true
        "diacritics": true
        "stop-words": true
    }
    "plugins": {
        "@textlint/text": {
            "extensions": [".org"]
	}
-   }
}
```

**Comment by billywade** [comment]
It very well could be, I'm not very heavy as a JS guy, but if it is, it's that there's a problem with a language intricacy, not an unmatched paren or something like that.

**Comment by azu** [comment]
Well, `,` is missing?

```
{
    "rules": {
        "write-good": true
        "no-start-duplicated-conjunction": true
        "max-comma": true
        "terminology": true
        "period-in-list-item": true
        "abbr-within-parentheses": true
        "alex": true
        "common-misspellings": true
        "en-max-word-count": true
        "diacritics": true
        "stop-words": true
    }, // <=
    "plugins": {
        "@textlint/text": {
            "extensions": [".org"]
	    }
    }
}
```

**Comment by billywade** [comment]
Good catch, but I added it in and no dice.
